osu.Game.Rulesets.Taiko/Difficulty/TaikoPerformanceCalculator.cs

```diff
@@ -112,7 +113,14 @@ private double computeDifficultyValue(ScoreInfo score, TaikoDifficultyAttributes
             difficultyValue *= Math.Pow(missPenalty, countMiss);
 
             if (score.Mods.Any(m => m is ModHidden))
-                difficultyValue *= (isConvert) ? 1.025 : 1.1;
+            {
+                double hiddenBonus = (isConvert) ? 0.025 : 0.1;
```

Comment:
```suggestion
                double hiddenBonus = isConvert ? 0.025 : 0.1;
```

Brackets are redundant here